Opserver/Views/SQL/Operations.Top.Detail.cshtml

```diff
@@ -159,14 +159,14 @@
                     <div>
                         @{
                             var truncated = plan.Statements.Any(st => st.StatementText != null && st.StatementText.Length > 3998);
-                            var lastStatement = plan.Statements.Last();
+                            var lastStatement = plan.Statements.Any() ? plan.Statements.Last() : null;
```

Comment:
Ah ! I missed it. Thank you. `LastOrDefault()` wins. Ignore the method name in my commit message :). I am still recovering from the depression caused by #130